### PR TITLE
Update UI - issue #37

### LINKED ISSUE #37: Build basic UI
**Description**
Update the views (Auth pages, Tabs) to represent what we have in the storyboard

**Deliverables**
UI should look like storyboard implementation

### PR DESCRIPTION
**Pull request by akchen1**
I updated most of the UI. Just missing some icons and logos. @blchelle Can you add the welcome logo, back button, and book tab icon to the drawables? Right now I am using placeholders for those

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -27,6 +27,7 @@ dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.2'
+    implementation 'androidx.appcompat:appcompat:1.0.0'
```
Comment (mingyaang) [inline app/build.gradle]: what the, why we have two different veersions of appcompat

File: app/src/androidTest/java/com/team41/boromi/BookDBTest.java [hunk 2]
```diff
@@ -1,122 +1,123 @@
...
-		// Deletes a book from firestore
-		boolean deleteSuccessful = bookDB.deleteBook(testBooks.get(1).getBookId());
-		assertTrue(deleteSuccessful);
-
-		// Gets the updated books from firestore to ensure the count was decremented
-		ownedBooks = bookDB.getUsersOwnedBooks("testuser1");
-		assertEquals(1, ownedBooks.size());
-	}
-}
+package com.team41.boromi;
```
Comment (mingyaang) [inline app/src/androidTest/java/com/team41/boromi/BookDBTest.java:1]: why is github saying you changed this, I don't think you meant to commit this file

File: app/src/main/java/com/team41/boromi/BookActivity.java [hunk 3]
```diff
@@ -73,4 +80,24 @@ public void onTabReselected(Tab tab) {
       }
     });
   }
+
+  @Override
+  public boolean onCreateOptionsMenu(Menu menu) {
```
Comment (mingyaang) [inline app/src/main/java/com/team41/boromi/BookActivity.java:82]: due to crunch time, I'm gonna trust you this works andrew

File: app/src/main/java/com/team41/boromi/auth/LoginFragment.java [hunk 4]
```diff
@@ -14,6 +16,7 @@
 
 import com.team41.boromi.BookActivity;
 import com.team41.boromi.R;
+import org.w3c.dom.Text;
```
Comment (mingyaang) [inline app/src/main/java/com/team41/boromi/auth/LoginFragment.java]: what is with you importing weird libraries, what does this one do

Comment (mingyaang) [reply app/src/main/java/com/team41/boromi/auth/LoginFragment.java]: Make sure to optimize imports before you PR incasee you add an unused library when coding

File: app/src/androidTest/java/com/team41/boromi/BookDBTest.java (hunk 2 above)
Comment (mingyaang) [reply app/src/androidTest/java/com/team41/boromi/BookDBTest.java:1]: Its okay though, it's only a newline

**Review by mingyaang — APPROVED** [review]
lgtm if you make the changes requested

File: app/src/androidTest/java/com/team41/boromi/BookDBTest.java (hunk 2 above)
Comment (akchen1) [reply app/src/androidTest/java/com/team41/boromi/BookDBTest.java:1]: idk mbmb

File: app/src/main/java/com/team41/boromi/BookActivity.java (hunk 3 above)
Comment (akchen1) [reply app/src/main/java/com/team41/boromi/BookActivity.java:82]: yes

File: app/src/main/java/com/team41/boromi/auth/LoginFragment.java (hunk 4 above)
Comment (akchen1) [reply app/src/main/java/com/team41/boromi/auth/LoginFragment.java]: yes done

**Pull request merged**